Added a scroll if check on the scrollTo directive

s10mcow opened the pull request:
We now have the abiliity to check if a scroll should be made or not based on a certain criteria.

Commit:
Added a scroll if check on the scrollTo directive

We now have the abiliity to check if a scroll should be made or not based on a certain criteria.

d-oliveros commented:
This is great. Care to update the README.md to let people know about this feature? :smile:

stenmuchow commented:
no problem!

Commit:
Added info for scrollIf option to the scrollTo directive

Updated the readme to include usage instructions for the scrollIf truthy check, as to allow the scroll to fire or not.

resendes commented:
++

fernahh commented:
Hey, @d-oliveros... you can merge it? :smile:
